### ISSUE TITLE
html-has-lang accessibility violations for error html files and series listings home page

### ISSUE THREAD

**Issue by RyanAJensen** [pulibrary/repecwp#187]
### Steps to recreate
Visit repecwp homepage and run the Axe dev tools

### URL
http://localhost:3000/

### Issues
<html> tags on these pages are missing "lang=en-US" attribute

### Severity
Serious

### WCAG Success Criterion
[WCAG 3.1.1 Language of Page](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-doc-lang-id.html) 

### WCAG Conformance level (A, AA, AAA)
A

### User impact
Screen readers use the lang attribute to determine the language of content and without it, users may hear the wrong pronunciations or intonations.

### Recommended fix
To solve this problem, you need to fix the following: The element does not have a lang attribute

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] WCAG Success Criterion are resolved
